ui/components/ui/currency-display/currency-display.stories.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { decimalToHex } from '../../../../shared/modules/conversion.utils';
+import CurrencyDisplay from '.';
+
+export default {
+  title: 'Components/UI/CurrencyDisplay',
+
+  argTypes: {
+    value: {
+      control: {
+        type: 'text',
+      },
+    },
+    displayValue: {
+      control: {
+        type: 'text',
+      },
+    },
+    currency: {
+      control: {
+        type: 'text',
+      },
+    },
+    prefix: {
+      control: {
+        type: 'text',
+      },
+    },
+    suffix: {
+      control: {
+        type: 'text',
+      },
+    },
+    numberOfDecimals: {
+      control: {
+        type: 'text',
+      },
+    },
+    denomination: {
+      control: {
+        type: 'text',
+      },
+    },
+  },
```

Comment:
You could update the argTypes to include all props. You will need to import `import { EtherDenomination } from '../../../../shared/constants/common';`

```suggestion
argTypes: {
    className: {
      control: 'text',
    },
    currency: {
      control: 'text',
    },
    'data-testid': {
      control: 'text',
    },
    denomination: {
      control: 'select',
      options: [EtherDenomination.ETH, EtherDenomination.GWEI],
    },
    displayValue: {
      control: 'text',
    },
    hideLabel: {
      control: 'boolean',
    },
    hideTitle: {
      control: 'boolean',
    },
    numberOfDecimals: {
      control: 'number',
    },
    prefix: {
      control: 'text',
    },
    prefixComponent: {
      control: 'text',
    },
    style: {
      control: 'object',
    },
    suffix: {
      control: 'text',
    },
    value: {
      control: 'text',
    },
    prefixComponentWrapperProps: {
      control: 'object',
    },
    textProps: {
      control: 'object',
    },
    suffixProps: {
      control: 'object',
    },
    boxProps: {
      control: 'object',
    },
  },
```